gcovr/__main__.py

```diff
@@ -61,10 +61,25 @@ def fail_under(covdata, threshold_line, threshold_branch):
         percent_branches = 100.0
 
     if percent < threshold_line and percent_branches < threshold_branch:
+        logger.error(
+            "Failed, not enough line coverage (got: {}, expected min: {})",
+            percent,
+            threshold_line
+        )
```

Comment:
Both the line and branch coverage failed the threshold, but only line coverage is mentioned. Is that intentional?

I'd probably write this code like this, so that independent error messages are shown for each criterion:

```python
exit_code = 0
if percent < threshold_line:
    exit_code |= 2
    logger.error("failed minimum line coverage (got {}%, minimum {}%)", percent, threshold_line)
if percent_branches < threshold_branch:
    exit_code |= 4
    logger.error("failed minimum branch coverage (got {}%, minimum {}%)", percent_branches, threshold_branch)
if exit_code:
    sys.exit(exit_code)
```

Reply:
You'r right but the bitwise or for the exitcode is confusing for me. If you search for the sys.ext(6) you'll never find it. What about doing something like:
```python
line_nok = False
branch_nok = False
if percent < threshold_line:
    line_nok = True
    logger.error("failed minimum line coverage (got {}%, minimum {}%)", percent, threshold_line)
if percent_branches < threshold_branch:
    branch_nok = True
    logger.error("failed minimum branch coverage (got {}%, minimum {}%)", percent_branches, threshold_branch)
if line_nok and branch_nok:
    sys.exit(6)
if line_nok:
    sys.exit(2)
if branch_nok:
    sys.exit(4)
```